heron/common/src/cpp/network/network_error.h

```diff
@@ -30,4 +33,12 @@ enum NetworkErrorCode {
   TIMEOUT
 };
 
+static inline std::ostream& operator<<(std::ostream& os, const NetworkErrorCode code) {
+  std::string strs[] = { "OK", "Connection error", "Read error", "Write error",
```

Comment:
I am not aware of any C++ syntax for this, we may archive it by a wrapper class or something like that, but I doubt it is worth any effort to improve it. Does it hurt readability to you?